src/Components/Web.JS/src/Platform/Circuits/DefaultReconnectDisplay.ts

```diff
@@ -77,11 +77,13 @@ export class DefaultReconnectDisplay implements ReconnectDisplay {
     this.button.style.display = 'block';
     this.reloadParagraph.style.display = 'none';
     this.message.innerHTML = 'Reconnection failed. Try <a href>reloading</a> the page if you\'re unable to reconnect.';
+    this.message.querySelector('a')!.addEventListener('click', () => location.reload());
```

Comment:
Maybe also add `e.preventDefault()`.
Because `location.reload()` also keeps the url fragment `#` whereas just the `href` does not.

Reply:
We want to keep the whole URL. The `href` is not involved because normal navigation can't occur while disconnected.